Bastion/src/isaac/bastion/listeners/CommandListener.java

```diff
@@ -1,92 +0,0 @@
-package isaac.bastion.listeners;
-
-import isaac.bastion.Bastion;
-import isaac.bastion.BastionBlock;
-import isaac.bastion.commands.PlayersStates;
-import isaac.bastion.commands.PlayersStates.Mode;
-import isaac.bastion.manager.BastionBlockManager;
-
-import org.bukkit.ChatColor;
-import org.bukkit.Material;
-import org.bukkit.block.Block;
-import org.bukkit.entity.Player;
-import org.bukkit.event.EventHandler;
-import org.bukkit.event.Listener;
-import org.bukkit.event.block.Action;
-import org.bukkit.event.player.PlayerInteractEvent;
-
-import vg.civcraft.mc.citadel.Citadel;
-import vg.civcraft.mc.citadel.reinforcement.PlayerReinforcement;
-
-public class CommandListener implements Listener {
-	private static BastionBlockManager manager;
-	private Material bastionBlock = Bastion.getConfigManager().getBastionBlockMaterial();
-	
-	public CommandListener() {
-		manager=Bastion.getBastionManager();
-	}
-	
-	@EventHandler(ignoreCancelled=true)
-	public void clicked(PlayerInteractEvent event) {
```

Comment:
I thought the name CommandListener didn't make sense so this is now BastionInteractListener, maybe BastionCreateListener would be a better name idk